build/build-dotnet.sh

```diff
@@ -41,10 +41,36 @@ DIR=$(pwd)/dotnet/runtime
 git clone --depth 1 -b ${BRANCH} ${URL} ${DIR}
 cd ${DIR}
 
-./build.sh clr -rc Checked --ninja
 
-mv .dotnet/ artifacts/bin/coreclr/Linux.x64.Checked/
-XZ_OPT=-2 tar Jcf ${OUTPUT} artifacts/bin/coreclr/Linux.x64.Checked
+CORE_ROOT=artifacts/tests/coreclr/Linux.x64.Release/Tests/Core_Root
+
+# Build everything in Release mode
+./build.sh Clr+Libs -c Release --ninja
+
+# Build Checked JIT compilers (only Checked JITs are able to print codegen)
+./build.sh Clr.Runtime -c Checked --ninja
```

Comment:
```suggestion
./build.sh Clr.AllJits -c Checked --ninja
```

You can speed up this even more - if you build only JITs.

Reply:
Yeah but I was not sure it worked 🙂